lib/index.js

```diff
@@ -171,7 +171,7 @@ const create = (client, fs) => {
                 });
 
                 return decrypted.map(pair => {
-                  return `export ${pair.key}=${pair.value};echo "Decrypted ${pair.key}";`;
+                  return `export ${pair.key}="${pair.value}";echo "Decrypted ${pair.key}";`;
```

Comment:
Thanks for looking into this.  This seems to be similar to the issue #5 that i've filed a month or so ago. However, it does not address additional cases. For example if the secure value has a bash special character in it, we also get issues. 

Example,  MY_SECURE_VAR=blah&blah

even if we wrap this in double quotes, it would cause a failure.

Reply:
scratch that. tested with double quotes. seems okay